backend/tests/core/Entities/ProjectHelper.cs

```diff
@@ -79,8 +79,9 @@ public static List<Entity.Project> CreateProjects(int startId, int count, Entity
         /// <param name="id"></param>
         /// <param name="agency"></param>
         /// <returns></returns>
-        public static Entity.Project CreateProject(this PimsContext context, int id, Entity.Agency agency)
+        public static Entity.Project CreateProject(this PimsContext context, int id, int agencyId)
```

Comment:
You probably want to add an override rather than change the function structure here.

Reply:
The reason for this change was actually to prevent a stack overflow, the above method calls itself infinitely.